docs/install/install-from-packages.md

```diff
@@ -37,6 +43,15 @@ You may host multiple Plone sites on the same server.
 
 ### Pre-requisites for installation
 
+-   An operating system that runs all the pre-requisites.
+    Most UNIX-based operating systems are supported, including many Linux distributions, macOS, or WSL on Windows.
```

Comment:
WSL should probably be a link or glossary term, no? Imagine I'm a Windows user who does not know what WSL is; where do I go next?

Reply:
Fixed in 3bcdc9665ca2316216807fc3fb18dee00bb1b8d6. Thank you!